r/readahead/readahead_ubi8.4.sh

```diff
@@ -0,0 +1,60 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : github.com/klauspost/readahead
```

Comment:
This field is for package name, in this case, readahead. Please update.

Reply:
done